model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -127,7 +127,7 @@ data class ResolvedLicenseInfo(
         return if (choices.isEmpty()) {
             resolvedLicenses
         } else {
-            resolvedLicenses?.applyChoices(choices)?.validChoices()?.reduceOrNull(SpdxExpression::or)
+            resolvedLicenses?.applyChoices(choices)?.validChoices()?.toExpression(SpdxOperator.OR)
```

Comment:
commit-msg: Should we link https://github.com/oss-review-toolkit/ort/pull/9909 as it is analog?